packages/notifi-frontend-client/lib/models/FusionEvent.ts

```diff
@@ -0,0 +1,53 @@
+export type FusionEventMetadata = {
+  uiConfigOverride?: {
+    topicDisplayName?: string;
+    historyDisplayName?: string;
+  };
+  filters: Array<AlertFilter>;
+  requiredParserVariables: Array<RequiredParserVariable>;
+};
+
+/**
+ * @param name - `string` unique name
+ */
+export type AlertFilter = {
+  name: string;
+  type: AlertFilterType;
+  executionPriority: number;
+  minimumDurationBetweenTriggersInMinutes: number;
```

Comment:
Need double check if this (`minimumDurationBetweenTriggersInMinutes`) should be optional or not? 🙇🏻
CC: @nimesh-notifi @mayuranganesathas @jamie-notifi

Reply:
just saw this. These questions were all resolved, right?

Reply:
minimumDurationBetweenTriggersInMinutes should be removed from the base AlertFilter

Reply:
Modify the `AlertFilter` type accordingly.